jobs/metricsgateway/spec

```diff
@@ -35,50 +35,50 @@ properties:
     description: "the level of logging"
     default: "info"
   autoscaler.metricsgateway.envelop_chan_size:
-    description: ""
+    description: "the channel size to transport envelope from nozzles to emitters"
     default: 500
   autoscaler.metricsgateway.nozzle_count:
-    description: ""
+    description: "the number of go-loggregator client in a metricsgateway"
     default: 3
   autoscaler.metricsgateway.app_manager.app_refresh_interval:
-    description: ""
+    description: "the interval to fresh apps in cache from policy db"
     default: 5s
 
   autoscaler.metricsgateway.emitter.buffer_size:
-    description: ""
+    description: "the size of channel in a emmiter to store envelopes waiting to send to metricsserver"
     default: 500
   autoscaler.metricsgateway.emitter.keep_alive_interval:
-    description: ""
+    description: "keep alive interval of the websocket between emitter and metricsserver"
```

Comment:
the time interval of keep alive  for the websocket between emitter and metrics server